src/frontend/komponenter/Fagsak/Vilkår/BehandleVilkårSkjema.tsx

```diff
@@ -73,8 +72,18 @@ const BehandlingVilkårSkjema: React.FunctionComponent<IBehandlingVilkårSkjema>
                     'Du må velge et behandlingsresultat'
                 }
             />
+            {behandlingstype === Behandlingstype.REVURDERING && context.behandlingResultat === BehandlingResultat.AVSLÅTT && (
+                <div className={'vilkår__skjemagruppe'}>
+                    <Undertittel children={'Opphør utbetalinger for fagsak'}/>
```

Comment:
Trenger man undertittel her?

Kanskje det holder med informasjonsbolk med `Forventet opphørsmåned: inneværendeMåned`.
Greit med en avsjekk med Anna og/eller Sela på hvordan det blir? Eller er det kanskje bare for oss uansett?